alerts/cluster-monitoring-operator/PrometheusRuleFailures.md

```diff
@@ -0,0 +1,36 @@
+# PrometheusRuleFailures
+
+## Meaning
+
+The `PrometheusRuleFailures` alert triggers when there has been
+a constant increase in failed evaluations of prometheus rules for more 
+than 15 minutes.
+
+## Impact
+
+Metrics and alerts might be missing or inaccurate.
+
+## Diagnosis
+
+1. Determine whether the alert has triggered for the instance of Prometheus used
+for default cluster monitoring or for the instance that monitors user-defined
+projects by viewing the alert message's `namespace` label: the namespace for
+default cluster monitoring is `openshift-monitoring`; the namespace for user
+workload monitoring is `openshift-user-workload-monitoring`.
+
+1. Review the logs for the affected Prometheus instance:
```

Comment:
I haven't checked in details but the alert might contain labels that could help identify the faulty rule.

Reply:
At least looking at the alert definition on Prometheus it didn't seem like that was the case, not sure if I might be missing something else.

Reply:
@JoaoBraveCoding I made a few suggested changes to the language. PTAL when you get a chance and lmk if you have any questions. Sorry for my delayed response!